src/redux/books/books.js

```diff
@@ -8,27 +14,56 @@ export const addBook = (book) => ({
   payload: book,
 });
 
-const initialState = [{
-  id: 1,
-  title: 'Benjamin biography',
-  author: 'Benkis',
-},
-{
-  id: 2,
-  title: 'L ame du monde',
-  author: 'Frederic francois',
-},
+export const read = (books) => ({
+  type: 'READ_BOOK',
+  payload: books,
+});
+
+const initialState = [
+//   id: 1,
+//   title: 'Benjamin biography',
+//   author: 'Benkis',
+// },
+// {
+//   id: 2,
+//   title: 'L ame du monde',
+//   author: 'Frederic francois',
+// },
```

Comment:
### OPTIONAL

- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉